deps/build.jl

```diff
@@ -16,6 +16,55 @@ function library_name(v)
     return "$(cpx_prefix)cplex$(v).$(Libdl.dlext)"
 end
 
+function possible_paths(cplex_studio_paths::Vector{String})
+    return map(cplex_studio_paths) do f
+        if Sys.iswindows()
+            return escape_string("C:\\Program Files\\IBM\\ILOG\\$f\\cplex\\bin\\x64_win64\\")
+        elseif Sys.isapple()
+            return "/Applications/$f/cplex/bin/x86-64_osx/"
+        else
+            return "/opt/$f/cplex/bin/x86-64_linux/"
+        end
+    end
+end
+
+function guess_cplex_folder(cplex_studio_paths::Vector{String})
+    for f in possible_paths(cplex_studio_paths)
+        if isdir(f)
+            return true, f
+        end
+    end
+
+    return false, nothing
+end
+
+function get_error_message_if_not_found()
+    return """
+    Unable to install CPLEX.jl.
+
+    The versions of CPLEX supported by CPLEX.jl are:
+
+    * 12.10
+    * 20.1
+
+    You must download and install one of these versions separately.
+
+    You should set the `CPLEX_STUDIO_BINARIES` environment variable to point to
+    the install location then try again. For example (updating the path to the
+    correct location if needed):
+    
+    ```
+    ENV["CPLEX_STUDIO_BINARIES"] = "$(possible_paths(["CPLEX_Studio201"]))"
```

Comment:
`$(possible_path("CPLEX_Studio201")`

Reply:
The problem is that `possible_path` takes a vector of possibilities in argument. Here, only the latest one is shown. Making it work with strings would be more complicated, as `map` would iterate over the characters.

Reply:
I'm suggesting to also modify `possible_path` to take a single string and return a single string.

Reply:
Indeed, it's no more required to have a vector here. Better this way?